modules/fedimint-wallet/src/lib.rs

```diff
@@ -353,16 +353,14 @@ impl FederationModule for Wallet {
                 output.recipient.network,
             ));
         }
-        let consensus_fee_rate = self.current_round_consensus().unwrap().fee_rate;
+        let consensus_fee_rate = self.current_round_consensus()?.fee_rate;
```

Comment:
I think there should always be a round consensus (except maybe for epoch 0? need to check) in the DB, so this should be an `expect` instead of `unwrap` explaining why this is the case.

Reply:
I think I misunderstand what our goal is: I thought we want to remove `unwraps/expects` completely  and only if not possible otherwise use an `expect`. Even though we expect that there should always be a round consensus, having an error for that wouldn't hurt or not ?